views/js/admin/src/components/common/PsCard.vue

```diff
@@ -0,0 +1,34 @@
+<template>
+  <div class="card">
+    <LoaderOverlay :show="loading" />
+    <div
+      v-if="$slots.header"
+      class="card-header">
+      <slot name="header" />
+    </div>
+    <div class="card-body">
+      <slot />
+    </div>
+    <div
+      v-if="$slots.footer"
+      class="card-footer d-flex">
+      <slot name="footer" />
+    </div>
+  </div>
+</template>
+
+<script lang="ts">
+import LoaderOverlay from '@/components/common/LoaderOverlay.vue';
+import { defineComponent } from '@vue/composition-api';
+
+export default defineComponent({
+  name: 'PsCard',
+  components: { LoaderOverlay },
+
+  props: {
+    loading: {
+      type: Boolean,
```

Comment:
required / default value?

Reply:
Hoeft niet bij een boolean. Deze is optioneel met een default van `false`